src-built-in/components/fileMenu/src/stores/fileMenuStore.js

```diff
@@ -88,6 +90,33 @@ var Actions = {
 	hideWindow() {
 		FileMenuStore.finWindow.hide();
 	},
+	//If a scheduled restart is invoked, we will open a dialog that gives the user a chance to cancel the restart.
+	listenForScheduledRestart() {
```

Comment:
@carterbs FileMenu isn't the place for this. We've got too much hard coded crap in FileMenu as it is, let's not add to the mess. This code should be in the service.